### PR TITLE
add action to view stats of already uploaded changesets

### PR DESCRIPTION
**Pull request by brianhatchl**
refs #1829

### CODE REVIEW

File: modules/Hoot/ui/modals/grailDatasetPicker.js [hunk 1]
```diff
@@ -56,8 +56,10 @@ export default class GrailDatasetPicker {
         let formId = 'grailDatasetForm';
         this.form  = this.formFactory.generateForm( 'body', formId, metadata );
 
-        this.submitButton = d3.select( `#${ metadata.button.id }` );
-        this.updateSubmitButton();
+        if (this.refDatasets.length) {
+            this.submitButton = d3.select( `#${ metadata.button.id }` );
+            this.updateSubmitButton();
+        }
```
Comment (brianhatchl) [inline modules/Hoot/ui/modals/grailDatasetPicker.js:62]: fixes js error when no suitable grail datasets found
```
grailDatasetPicker.js:59 Uncaught (in promise) TypeError: Cannot read property 'id' of undefined
    at GrailDatasetPicker.render (grailDatasetPicker.js:59)
```

File: modules/Hoot/ui/modals/changesetStats.js [hunk 2]
```diff
@@ -2,9 +2,10 @@ import FormFactory from '../../tools/formFactory';
 import { uiChangesetEditor } from '../../../ui/changeset_editor';
 
 export default class ChangesetStats {
-    constructor( job, data ) {
+    constructor( job, data, viewOnly ) {
```
Comment (brianhatchl) [inline modules/Hoot/ui/modals/changesetStats.js:5]: adds a view only mode with just stats table

**Comment by brianhatchl** [comment]
image

view only action

File: modules/Hoot/ui/modals/changesetStats.js [hunk 3]
```diff
@@ -56,27 +57,28 @@ export default class ChangesetStats {
...
-            title: titleText,
-            button: {
-                text: 'Upload Changeset',
-                id: 'SubmitBtn',
-                onClick: () => this.handleSubmit()
-            }
+            title: titleText
+        };
+
+        if (!this.viewOnly) metadata.button = {
```
Comment (bobi7) [inline modules/Hoot/ui/modals/changesetStats.js]: let's add `{`'s around this fancy man

**Pull request merged**